samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineTextSentimentAnalysisSample.java

```diff
@@ -0,0 +1,239 @@
+import com.google.cloud.aiplatform.v1beta1.EnvVar;
+import com.google.cloud.aiplatform.v1beta1.ExplanationMetadata;
+import com.google.cloud.aiplatform.v1beta1.ExplanationParameters;
+import com.google.cloud.aiplatform.v1beta1.ExplanationSpec;
+import com.google.cloud.aiplatform.v1beta1.FilterSplit;
+import com.google.cloud.aiplatform.v1beta1.FractionSplit;
+import com.google.cloud.aiplatform.v1beta1.InputDataConfig;
+import com.google.cloud.aiplatform.v1beta1.LocationName;
+import com.google.cloud.aiplatform.v1beta1.Model;
+import com.google.cloud.aiplatform.v1beta1.Model.ExportFormat;
+import com.google.cloud.aiplatform.v1beta1.ModelContainerSpec;
+import com.google.cloud.aiplatform.v1beta1.PipelineServiceClient;
+import com.google.cloud.aiplatform.v1beta1.PipelineServiceSettings;
+import com.google.cloud.aiplatform.v1beta1.Port;
+import com.google.cloud.aiplatform.v1beta1.PredefinedSplit;
+import com.google.cloud.aiplatform.v1beta1.PredictSchemata;
+import com.google.cloud.aiplatform.v1beta1.SampledShapleyAttribution;
+import com.google.cloud.aiplatform.v1beta1.TimestampSplit;
+import com.google.cloud.aiplatform.v1beta1.TrainingPipeline;
+import com.google.protobuf.Any;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import com.google.rpc.Status;
+import java.io.IOException;
+import java.util.List;
+
+public class CreateTrainingPipelineTextSentimentAnalysisSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String trainingPipelineDisplayName = "YOUR_TRAINING_PIPELINE_DISPLAY_NAME";
+    String project = "YOUR_PROJECT_ID";
+    String datasetId = "YOUR_DATASET_ID";
+    String modelDisplayName = "YOUR_MODEL_DISPLAY_NAME";
+
+    createTrainingPipelineTextSentimentAnalysisSample(
+        project, trainingPipelineDisplayName, datasetId, modelDisplayName);
+  }
+
+  static void createTrainingPipelineTextSentimentAnalysisSample(
+      String project, String trainingPipelineDisplayName, String datasetId, String modelDisplayName)
+      throws IOException {
+    PipelineServiceSettings pipelineServiceSettings =
+        PipelineServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (PipelineServiceClient pipelineServiceClient =
+        PipelineServiceClient.create(pipelineServiceSettings)) {
+      String location = "us-central1";
+      String trainingTaskDefinition =
+          "gs://google-cloud-aiplatform/schema/trainingjob/definition/"
+                  + "automl_text_sentiment_1.0.0.yaml";
+
+      // Use sentimentMax of 4
```

Comment:
Is this a recommendation or just a description? If it is a recommendation, can we add a bit more context on why we should use sentimentMax of 4? If it's a description then can we link to sentimentMax doc?

Reply:
I just added detailed explanation on sentiment max